### PR TITLE
Update landing page queries and urls

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: static/js/apps/explore_landing/topics.json [hunk 1]
```diff
@@ -461,94 +377,66 @@
...
-            "url": "p=geoId/06&t=Count_Person_EducationalAttainmentBachelorsDegree&pt=County&oq=Bachelors+degree+holders+in+California+counties"
-          },
-          {
-            "title": "US counties with the most number of PhDs",
-            "url": "p=country/USA&t=Count_Person_EducationalAttainmentDoctorateDegree&pt=County&oq=US+counties+with+the+most+number+of+PhDs"
-          },
-          {
-            "title": "Home value in Detroit",
-            "url": "p=geoId/2622000&t=dc/topic/HousesByHomeValue&oq=Home+value+in+Detroit"
+            "title": "which US county has the most number of PhDs",
```
Comment (pradh) [inline static/js/apps/explore_landing/topics.json:404]: capitalize

File: static/js/apps/explore_landing/topics.json [hunk 2]
```diff
@@ -141,106 +137,66 @@
...
           {
-            "title": "GDP of Iran",
-            "url": "p=country/IRN&t=dc/topic/GDP&oq=GDP+of+Iran"
+            "title": "Tell me about the economy in Brazil",
+            "url": "oq=Tell+me+about+the+economy+in+Brazil"
           },
           {
-            "title": "Remittances in Lebanon",
-            "url": "p=country/LBN&t=dc/topic/Remittance&oq=Remittances+in+Lebanon"
+            "title": "What is GDP of Iran",
```
Comment (pradh) [inline static/js/apps/explore_landing/topics.json:164]: nit: What is *the* GDP of Iran

File: static/js/apps/explore_landing/topics.json [hunk 3]
```diff
@@ -602,78 +490,66 @@
...
           {
-            "title": "Gender distribution in the US",
-            "url": "p=country/USA&t=dc/topic/Gender&oq=Gender+distribution+in+the+US"
+            "title": "What is the racial distribution of the US",
+            "url": "oq=What+is+the+racial+distribution+of+the+US"
           },
           {
-            "title": "Languages spoken in California",
-            "url": "p=geoId/06&t=dc/topic/Language&oq=Languages+spoken+in+California"
+            "title": "What languages spoken at home in California",
```
Comment (pradh) [inline static/js/apps/explore_landing/topics.json:497]: What languages *are* spoken ...

**Review by pradh — APPROVED** [review]
Thanks!

File: static/js/apps/explore_landing/topics.json (hunk 2 above)
Comment (shifucun) [reply static/js/apps/explore_landing/topics.json]: Done

File: static/js/apps/explore_landing/topics.json (hunk 1 above)
Comment (shifucun) [reply static/js/apps/explore_landing/topics.json]: Done

File: static/js/apps/explore_landing/topics.json (hunk 3 above)
Comment (shifucun) [reply static/js/apps/explore_landing/topics.json]: Done

**Pull request merged**